### DISCUSSION TITLE
I am attempting to setup the medusa codebase, but the 'yarn build' is failing for some reason. I need help

### DISCUSSION THREAD

**Discussion by GetSetAbhi** [medusajs/medusa#5072] [Q&A]
node version : 18.17.1
yarn version : 3.2.1
operation system : ubuntu

I am not able to successfully execute 'yarn build' while attempting to setup the codebase. I need help please

![image](https://github.com/medusajs/medusa/assets/5512947/263a73bc-7db8-45b0-8994-e6865e4632e2)

**Comment by pevey** [comment]
I'm curious, what is your node version?

```
node -v
```
